app/models/course.rb

```diff
@@ -37,4 +37,16 @@ def university_based?
   def further_education?
     level == 'further_education' && subjects.any? { |s| s.subject_name == 'Further education' || s.subject_code = '41' }
   end
+
+  def travel_to_work_areas
+    travel_to_work_areas = sites.map { |site| site.london_borough || site.travel_to_work_area }.uniq
+    return if travel_to_work_areas.blank?
+
+    if travel_to_work_areas.length == 1
+      travel_to_work_areas.first
+    else
+      last_travel_to_work_area = travel_to_work_areas.pop
+      travel_to_work_areas.join(', ') + " and #{last_travel_to_work_area}"
```

Comment:
When you pop the last value off the array it should return as a string so you don't need to interpolate, wrote mine like this in the end.
```suggestion
      travel_to_work_areas.join(', ') + " and " + last_travel_to_work_area
```